apps/aehttp/src/aehttp_api_handler.erl

```diff
@@ -63,8 +63,8 @@ handle_request_json(Req0, State = #state{
             Req = cowboy_req:reply(400, #{}, Body, Req1),
             {stop, Req, State}
     catch error:Error ->
-            lager:error("Unexpected validate result: ~p / ~p",
```

Comment:
It seems to me that this shall be kept as strictly-more-than-info log, on the basis that unidentified error shall be notified to the operator as maybe client request is valid but validator in server handler is incorrect as not catering for case.